en/user_guide/media.rst

```diff
@@ -0,0 +1,60 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel.
+
+Folder
+------
+
+All files are organised inside folders. It's possible to browse the folder tree to see files by clicking on the left panel.
+
+Create
+~~~~~~
+
+Creating a folder is done by clicking on the "new folder" button in the left panel.
+
+The creation form needs:
+
+* name: name of the folder
+* Websites: which websites are using the folder
+
+Remove
+~~~~~~
+
+To remove a folder, select the folder and click on the trash button on the right panel.
+
+Manage files
+------------
+
+The add button in the folder's content allows to upload files.
+
+A trash icon appears on media preview hovering to delete the media.
+
+Click on media preview open a page with two panels: crop files (for image) and add meta information.
+
+Image Crop
+~~~~~~~~~~
+
+When you upload an image, you can crop preview from some predetermined formats which can be personalized on custom installation:
+
+* original: keep original format
+* fixed height: adapt height and keep proportion
+* fixed width: adapt width and keep proportion
+* rectangle: reduce to fit in the rectangle format
+
+Once cropping is done the preview is updated and some choices are added:
+
+* manual crop: to adjust your selection manually
+* upload an alternative image: if user want to resize image himself, replace the selected thumbnail format by another image
```

Comment:
Rather than a custom resize this option is helpfull when the user want replace the original for this format.
For instance, if the original image is a HD one containing some texts, a little thumbnail won't be readable, so an alternative image more appropriatted to this little size can be uploaded.